package.json

```diff
@@ -17,17 +17,12 @@
     "tracks"
   ],
   "scripts": {
-    "start": "webpack-dev-server --config ./webpack.config.js --mode development --content-base app",
-    "build": "rm -rf build/*; rm -rf dist/*; mkdir -p dist/; webpack --config ./webpack.config.js --mode development; cp build/hglib.js dist/; cp app/index.html dist/; cp build/hglib.css dist/hglib.css;",
-    "build-amd": "rm -rf build/*; rm -rf dist/*; mkdir -p dist/; webpack --config ./webpack.config.amd.js --mode development; cp build/hglib-amd.js dist/; cp build/hglib-amd.css dist/hglib-amd.css;",
-    "build-amd-dev": "rm -rf build/*; rm -rf dist/*; mkdir -p dist/; webpack --config ./webpack.config.amd.dev.js --mode development; cp build/hglib-amd-dev.js dist/; cp build/hglib-amd-dev.css dist/hglib-amd-dev.css;",
-    "build-watch": "rm -rf build/*; rm -rf dist/*; mkdir -p dist/; webpack --config ./webpack.config.js --mode development --watch; cp build/hglib.js dist/; cp app/index.html dist/; cp build/hglib.css dist/hglib.css;",
+    "test": "wtr",
+    "start": "vite",
+    "build": "vite build",
     "compile": "rm -rf build/*; rm -rf dist/*; mkdir -p dist/; webpack --config ./webpack.config.js --mode production; cp build/hglib.min.js dist/; cp build/hglib.js dist/; cp app/index.html dist/; cp build/hglib.css dist/hglib.css; zip -r dist.zip dist",
```

Comment:
Should this be changed or removed? Looks like it still uses webpack.

Reply:
Fixed. I wrote `./scripts/compile.mjs` to minify the js output and generate an `index.html` which uses the UMD bundle.